pkg/systemd/hwinfo.jsx

```diff
@@ -192,173 +183,164 @@ class CPUSecurityMitigationsDialog extends React.Component {
         cockpit.script(kernelopt_sh, options, { superuser: "require", err: "message" })
                 .then(() => {
                     cockpit.spawn(["shutdown", "--reboot", "now"], { superuser: "require", err: "message" })
-                            .catch(error => this.setState({ rebooting: false, alert: error.message }));
+                            .catch(error => { setRebooting(true); setAlert(error.message) });
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-19378-20230922-161344-dd3ec708-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)